apps/hash-graph/lib/graph/src/knowledge/entity/query.rs

```diff
@@ -347,7 +347,7 @@ impl<'de> Visitor<'de> for EntityQueryPathVisitor {
 
         Ok(match token {
             EntityQueryToken::Uuid => EntityQueryPath::Uuid,
-            EntityQueryToken::RecordId => EntityQueryPath::RecordId,
+            EntityQueryToken::RecordId => EntityQueryPath::EditionId,
```

Comment:
`EntityQueryToken::RecordId` was not renamed as I avoided touching the REST API. `utoipa` does not understand `serde(alias)`, which would have been the preferred way.

Reply:
When are we going to touch the REST API then? 

I think I'd be more comfortable if these PRs actually renamed things across the stack in one go each time, rather than doing it in pieces where we have an inconsistent state of the branch again. I appreciate that we're trying to get DB changes stacked and ready, and I also appreciate that we're blocked on updating the whole branch until the BP stuff is in, so making TS changes now would just lead to fighting conflicts... I defer to your judgement, just wanted to flag my thoughts.